models/iaf_chxk_2008.nestml

```diff
@@ -27,7 +27,8 @@ neuron iaf_chxk_2008:
   initial_values:
     V_m mV = E_L   # membrane potential
     G_ahp nS = 0 nS      # AHP conductance
-    G_ahp' nS/ms = e / tau_ahp * nS   # AHP conductance
+    G_ahp' nS/ms = 0 nS/ms   # AHP conductance
+    #G_ahp' nS/ms = e / tau_ahp * nS   # AHP conductance
```

Comment:
why is this commented? if there is a reason for keeping it, then an explanatory comment would be nice, otherwise I think it should just be removed